modules/nw-ingress-configuring-application-domain.adoc

```diff
@@ -31,19 +31,21 @@ $ oc edit ingresses.config/cluster -o yaml
 +
 .. Edit the YAML file:
 +
-.Sample `appsDomain` configuration to `apps.acme.io`
+.Sample `appsDomain` configuration to `test.example.com`
 [source,yaml]
 ----
 apiVersion: config.openshift.io/v1
 kind: Ingress
 metadata:
   name: cluster
 spec:
-  domain: apps.<domain_url>
-  appsDomain: apps.acme.io
+  domain: apps.example.com            <1>
+  appsDomain: <test.example.com>      <2>
 ----
+<1> Default domain
+<2> Optional domain for AWS infrastructure to use. Instead of the default prefix, `apps`, you can use an alternative prefix like `test`.
```

Comment:
```suggestion
<2> Optional: Domain for AWS infrastructure to use for application routes. Instead of the default prefix, `apps`, you can use an alternative prefix like `test`.
```

Reply:
I'll take it since this is the point we're aiming to clarify in this PR fa sho.